Documentation/README_Pointers.md

```diff
@@ -1,76 +1,5 @@
-# Pointers #
+# Pointer #
```

Comment:
Because there may still be links out there referencing it. It's a pretty old file. Thus it's basically in a deprecated state and taken off the TOC until the next iteration. Just like all of the [Obsolete] tags